content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,112 @@
+
+## NetKVM VirtIO bringup
+NetKVM VirtIO is a NDIS 5 compatible driver developed by Red Hat Inc. which allows fast VirtIO networking in virtual machine software such as QEMU and other KVM hardware accelerated virtual machines.
+Nguyen Trung Khanh, a ReactOS contributor, imported the driver into the source tree of ReactOS and for this matter this heavily benefits the networking stack.
+
+## Miscellaneous changes & improvements
+Other areas of ReactOS infrastructure have received lots improvements, ranging from device drivers to software applications of ReactOS by various developers and contributors. Notably FreeLoader, the ReactOS' bootloader, received incremental fixes and further support improvements. As such Hermes Belusca-Maito added support for booting Linux 64-bit systems and fixed an issue where
+FreeLoader couldn't read from a non-BtrFS volume hence preventing booting. Dmitry Borisov provided fixed a serious triple fault when ReactOS was booted up in Screen mode. In addition to that, Dmitry Borisov also provided other fixes as well as adding ARC-emulation support necessary for NEC PC-98 series.
+Mark Harmstone also brought some fixes for the bootloader so that Windows Vista can be properly booted. On-Screen Keyboard and Accessbility Utility Manager are being regularly updated by Bisoc George and at the same time he provided some fixes or minor improvements to other parts of ReactOS.
+
+Herve Poussineau worked hard on ISA PnP (a driver component of PnP) which detects ISA devices although such driver is currently deactivated.
+Mark Jansen fixed a bug in comctl32 (Common Controls) module where Visual Basic 6 installer was rendered incorrectly. Eric Kohl worked on "Safely Remove Hardware" dialog manager where the peripherals are now enumerated for safely removal. A screenshot below describes
+the aforementioned dialog in question.
+
+{{< figure link="/img/project-news/reactos-0414-released/hotplug.png" src="/img/project-news/reactos-0414-released/hotplug.png" >}}
+
+What is particularly different between 0.4.13 and this release is the sleek performance due to obsolete fonts removed in ReactOS by Katayama Hirofumi MZ. This both reduced the binary size of this release and the RAM consumption.
+Here is a comparison between 0.4.13 (with the obsolete fonts used and loaded in memory) and 0.4.14.
+
+{{< figure link="/img/project-news/reactos-0414-released/0413-taskmgr.png" src="/img/project-news/reactos-0414-released/0413-taskmgr.png" >}}
+{{< figure link="/img/project-news/reactos-0414-released/0414-taskmgr.png" src="/img/project-news/reactos-0414-released/0414-taskmgr.png" >}}
+
+The 64-bit ReactOS port is extensively being worked on by Timo Kreuzer with continuous work on interrupt handling and other hardware issues like USB and networking issues in x64 build.
+Stanislav Motylkov still continues on working on Xbox boot support ranging from fixes patches and other updates brought by him.
+
```

Comment:
Some other things from me (just pick what you prefer from this list):

- Security-related commits:
```
commit 7e2db7733852804b65ecc756f09727dfc8f7cce4
[CSRSRV] Improve validation of CSR API Message's capture buffers.
In CsrCaptureArguments() and CsrReleaseCapturedArguments().
By implementing the checks done by Windows 2003 (NT 5.2) described
in section "Server-Side Validation and Capture" of the article
https://www.geoffchappell.com/studies/windows/win32/csrsrv/api/apireqst/capture_header.htm
```
```
commit afb14bff33efedffa796df27fa8d288b1ee1c504
[BLUE] Add IOCTL buffers validation. Addendum to commit bfd8a848.

commit cf7981fe4e5c2cbdf62d93b6132085a3ad383881
[LSASRV] Add parameter validation in LsapSetLogonSessionData().

commit 8210396cb42420abc57f121d0c278dee8fbfc671
[MSV1_0] Extend parameter validation in LsaApLogonUserEx2() and MsvpChangePassword(); fix crash in LsaApLogonUserEx2() when LogonDomainName points to a NULL string.
```

- Some old bugs finally fixed:
```
commit 5887b170052db6af1ee86dc68f053d0c79d15028
[HALX86] Fix the "ASSERT(j < 32);" problem in HalpStoreAndClearIopm() encountered from time to time.
CORE-11921 CORE-13715 (Regression introduced by commit 2e1b82cf, r44841.)

commit e8df1a38bd09e0416a1de7006615abfd1c845c9c
[CONSRV] Fix a corruption of the CONSRV memory when converting read input events back to ANSI.
This fixes a 6-year old copy-paste bug introduced by commit b47d51c6 (r64004).
```

Reply:
- Few enhancements for /SCREEN debugging display resets:
Correctly resetting the display once the KDBG debugger in SCREEN mode is being exited, and fixes CORE-12149 
```
commit bfd8a84865b7571e5e6c949b654830373651da59
[BLUE] Improve initialization and interfacing with INBV. CORE-15901

commit 0ad65796bbb57be739840c6f666cd9f8f8a1efda
[WIN32K][VIDEOPRT] Improve initialization and interfacing with INBV.
```